experimental/dds/tree2/src/shared-tree-core/editManager.ts

```diff
@@ -326,6 +349,7 @@ export class EditManager<
 				change: c.change,
 				revision: c.revision,
 				sequenceNumber: metadata.sequenceNumber,
+				clientSequenceNumber: metadata.clientSequenceNumber,
```

Comment:
This is often "0". Let's omit it when it is 0 to save storage. If it is not present when reading, we will assume 0.